src/config/swagger.json

```diff
@@ -9,6 +9,13 @@
     "http",
```

Comment:
switch this to swagger 3: https://mermade.org.uk/openapi-converter or create the ticket to do that next sprint please

Reply:
ill be doing this next sprint but good call.

Reply:
https://github.com/overture-stack/lectern/issues/17